common/defines/00_defines.txt

```diff
@@ -355,9 +368,11 @@ NMarriage = {
 	SPOUSE_FINDER_RELEVANCE_ALLIANCE_POWER_MULTIPLIER = 1.3 # How much the alliance power is valued for the relevance sorting of a marriage
 	SPOUSE_FINDER_RELEVANCE_GENERAL_SKILL_MULTIPLIER = 1.3 # How much the total general skill is valued for the relevance sorting of a marriage
 	SPOUSE_FINDER_RELEVANCE_INFERTILE_MODIFIER = -50000 # Modifier applied to relevance sorting of a marriage if they are visibly infertile
-	SPOUSE_FINDER_RELEVANCE_INFERTILE_MODIFIER_FEMALE_AGE = 35 # At this age females will be considered old enough for the infertile modifier even if they are younger than the actual too old for children define, so we sort out those close to that define earlier
-	SPOUSE_FINDER_RELEVANCE_AGE_DIFFERENCE_MULTIPLIER = -10 # Multiplier on age difference applied to relevance sorting of a marriage if the age difference is greater than or equal to the below
-	SPOUSE_FINDER_RELEVANCE_AGE_DIFFERENCE_CUTOFF = 5 # The age difference between the two parties for the above modifier to be applied per year of difference
+	
+	#Warcraft
+	SPOUSE_FINDER_RELEVANCE_INFERTILE_MODIFIER_FEMALE_AGE = 30000 # At this age females will be considered old enough for the infertile modifier even if they are younger than the actual too old for children define, so we sort out those close to that define earlier
+	SPOUSE_FINDER_RELEVANCE_AGE_DIFFERENCE_MULTIPLIER = 0 # Multiplier on age difference applied to relevance sorting of a marriage if the age difference is greater than or equal to the below
+	SPOUSE_FINDER_RELEVANCE_AGE_DIFFERENCE_CUTOFF = 0 # The age difference between the two parties for the above modifier to be applied per year of difference
```

Comment:
I wouldn't change it because it looks like it forces characters to look spouses of the same age.